spark/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -249,6 +290,34 @@ public Statistics estimateStatistics() {
     return new Stats(sizeInBytes, numRows);
   }
 
+  @Override
+  public boolean enableBatchRead() {
+    return lazyCheckEnableBatchRead();
```

Comment:
All this method does is call another method. Do we need both or can we move the body of `lazyCheckEnableBatchRead` here?

Reply:
Done in my PR.